### ISSUE TITLE
Accessibility Access fails after updating MacOS Ventura to 3.1

### ISSUE THREAD

**Issue by ej-github** [debauchee/barrier#1855]
### What happened?

Barrier was working great on Monterey and the GA release of Ventura.  I just updated my MBP M1 to Ventura 3.1 and Barrier is stuck on the Accessibility Access screen for System Settings.  It is already enabled in there.  The only options are to Open System Settings, and Deny.
![image](https://user-images.githubusercontent.com/45047655/207625900-ce803b4a-d743-4ec8-8a2b-8efd735f9467.png)
System Settings screenshot:
![image](https://user-images.githubusercontent.com/45047655/207626056-7c2f705b-013a-4ca2-89c8-818b7a07f6cc.png)

### Version

v2.4.0

### If applicable, where did you install Barrier from?

Downloaded from official release.

### What OSes are you seeing the problem on? (Check all that apply)

macOS

### What OS versions are you using?

Ventura 3.1

**Comment by myd-shakepay** [comment]
Same issue here :(

**Comment by myd-shakepay** [comment]
@ej-github I got it to work by removing it from the accessibility list in system settings and then relaunching barrier + re-enabling.

**Comment by Ligthert** [comment]
The workaround by @mdewji-shakepay worked for me.

**Comment by ej-github** [comment]
> @ej-github I got it to work by removing it from the accessibility list in system settings and then relaunching barrier + re-enabling.

Thanks, that worked for me!  I was toggling it on/off and didn't see the remove below.
